examples/ex9.cpp

```diff
@@ -53,6 +54,54 @@ double inflow_function(const Vector &x);
 // Mesh bounding box
 Vector bb_min, bb_max;
 
+class DG_Solver : public Solver
+{
+private:
+   SparseMatrix &M, &K, A;
+   GMRESSolver solver;
```

Comment:
It would be good to use a more descriptive name, e.g. linearSolver. The current choice is also very close to the Solver class name. This goes for ex9p as well.

Reply:
Good idea. Changed this.